sys-devel/llvm/llvm-3.4.2-r100.ebuild

```diff
@@ -203,8 +197,8 @@ src_install() {
 	if ! use clang; then
 		rm "${WORKDIR}"/${PN}-3.4-manpages/clang.1 || die
 	else
-		for tool in clang{,++}{,-${PV}} ; do
-			dosym /usr/bin/${tool} /usr/bin/${CHOST}-${tool}
+		for tool in clang{,++} ; do
+			dosym ${tool} /usr/bin/${CHOST}-${tool}
```

Comment:
It's a relative symlink. Using absolute path for a symlink to the file in the same directory is just stupid.